tern/analyze/passthrough.py

```diff
@@ -87,7 +89,8 @@ def run_extension(image_obj, ext_string, redo=False):
         )
         return mgr.driver.execute(image_obj, redo)
     except NoMatches:
-        return None
+        logger.critical(errors.unrecognized_extension.format(ext=ext_string))
```

Comment:
Instead, you can do `msg = errors.unrecognized_extension.format(ext=ext_string)` and then `logger.critical(msg)`